src/test/functional/tests/buzzPage.spec.ts

```diff
@@ -16,12 +18,23 @@ test.describe("Buzz Page", () => {
     await loginPage.loginUser(adminUserTestData.userName, adminUserTestData.password);
   });
 
-  test("Test Case 1: Share file and Delete file", async ({ page }) => {
+  test.afterEach(async ({ page }) => {
+    buzzPage = new BuzzPage(page);
+    await buzzPage.deletePhotos()
+  });
+
+  test("Test 1: Post should be shared", async ({ page }) => {
     await buzzPage.navigateToSubPage(buzzPage.buzzPageButton);
-    await buzzPage.sharePhotos();
+    await buzzPage.sharePhotos(filePath);
     await page.reload();
     await expect(buzzPage.photoImg.first()).toBeAttached();
-    await buzzPage.deletePhotos();
-    await expect(buzzPage.noPostParagraph.first()).toHaveText(buzzPageAssertions.noPostAvailable);
+  });
+
+  test("Test 2: Post should be edited", async ({ page }) => {
```

Comment:
OK. And now let's think of it test as a part of a bigger test suite running in parallel...